server_addon/houdini/server/settings/publish.py

```diff
@@ -44,6 +44,44 @@ class CollectLocalRenderInstancesModel(BaseSettingsModel):
     )
 
 
+def product_types_enum():
+    return [
+        {"value": "camera", "label": "Camera (Abc)"},
+        {"value": "pointcache", "label": "PointCache (Abc)/PointCache (Bgeo)"},
+        {"value": "review", "label": "Review"},
+        {"value": "staticMesh", "label": "Static Mesh (FBX)"},
+        {"value": "usd", "label": "USD (experimental)"},
+        {"value": "vdbcache", "label": "VDB Cache"},
+        {"value": "imagesequence", "label": "Composite (Image Sequence)"},
+        {"value": "ass", "label": "Arnold ASS"},
+        {"value": "arnold_rop", "label": "Arnold ROP"},
+        {"value": "mantraifd", "label": "Mantra IFD"},
```

Comment:
Mantra IFD publishing should be removed since https://github.com/ynput/ayon-core/pull/681

```suggestion
```